StyleGuide.md

```diff
@@ -5,98 +5,219 @@ The following guidance is provided to benefit CockroachDB docs authors and revie
+- **Avoid:** Each parameter should be set explicitly.  
+  **Prefer:** Set each parameter explicitly.
 
-- Recommended usage of the personal pronoun "we":
+- **Avoid:** Additional options can be specified.  
+  **Prefer:** You can specify additional options.
 
-    - "We" can be used to describe the group of people developing CockroachDB, instead of "Cockroach Labs," only when it is clear who "we" is referring to.
-    - Do not use "we" in place of "CockroachDB" for when you are talking about something the _product_ does or supports.
-    - Do not use "we" in tutorials. See the next bullet for more on tutorials and examples.
+Use concise, direct language. Cut unnecessary words unless a conversational tone is intentional (for example, in tutorials). If a feature or concept is difficult to describe clearly or concisely, consider using an [example](#examples) or [image](#images) to complement the text.
 
-- For tutorials and examples, we recommend you use the second-person point of view (e.g., you). These docs should be more casual and conversational, as if they are teaching the user, but still straightforward and clear.
+- **Avoid:** Be mindful of the possibility that you might encounter a different result, depending on the specifics of your configuration, so you might want to do some testing first to see what happens.  
+  **Prefer:** Run tests to verify that you get the expected result for your configuration.
 
-    **Example:** In this lab, you'll start with a fresh cluster, so make sure you've stopped and cleaned up the cluster from the previous labs.
+- **Avoid:** `table_name`: This parameter is used to specify the name of the table you want to modify.  
+  **Prefer:** `table_name`: The name of the table to modify.
 
-- Use active voice instead of passive. For more information, refer to the [Purdue Online Writing Lab resource](https://owl.english.purdue.edu/owl/resource/539/02/).
-- Use simple and direct language. Grammar can be incorrect to save simplicity (e.g., many descriptions in reference docs are phrases).
+Provide guidance rather than leave decisions to the user. Avoid vague phrases like "as you wish" or "of your choice". When user discretion is required, be as explicit as possible to reduce ambiguity and cognitive load.
 
-    **Example:** `table name`: The name of the table to create audit logs for.
+- **Avoid:** Increase the threshold as needed.  
+  **Prefer:** Increase the threshold until you see a performance improvement.
 
-- Avoid using "please" when giving an instruction, except when asking the user to go outside the scope of the task (such as contacting Cockroach Labs or filing a support issue).
+- **Avoid:** Define additional settings as desired.  
+  **Prefer:** You can define additional settings in the configuration file. To change the default host and port, set `host` and `port` respectively. To change the path, ...
```

Comment:
This implies that we should spell out every configuration option available, which can get unwieldy fast. Especially in deployment guides where we may want to offer a "one size fits all" initial configuration but leave room for optional configurations. In those circumstances I would rather link out if there's a good docs target or even leave out the sentences after the first generic one.

Reply:
https://www.cockroachlabs.com/docs/stable/deploy-cockroachdb-with-cockroachdb-operator#initialize-the-cluster Step 6 here is a hand-wavey note that essentially says "if you know you need to make other changes to your configuration yaml, now is a good time to make those" without going into details of every option. I believe it's compliant with the first sentence of your **Prefer** example, but it refers to only a couple example configs and doesn't have task-oriented sentences like the "To change the ... set ..." in that example.

I would just want to clarify in the style guide if this approach is acceptable when there are too many specific configs to spell out in that single instruction

Reply:
I see. The style guidance here is specifically intended to guardrail us against hand-waving. It's of course not possible to mention or prescribe a value for every configurable detail of every customer's deployment. But from past/ongoing experiences, phrases like "If you know you need ..." or "Modify other relevant parts of the configuration ..." will prompt at least some customers to respond, "How do I know I need to?" and "Which parts are relevant?". A subtle balancing act might be to simply state in a callout that "Depending on your configuration, you may need to set additional parameters in the XYZ fields". 

@Flat5flat9 wondering if you have any thoughts here.